index.c

```diff
@@ -1543,32 +1563,32 @@ int mutt_index_menu(void)
           menu->redraw = REDRAW_FULL;
 
         break;
+      }
 
         /* --------------------------------------------------------------------
          * 'index' specific commands
          */
 
       case OP_MAIN_DELETE_PATTERN:
+        if (!prereq(Context, menu, CHECK_IN_MAILBOX | CHECK_MSGCOUNT | CHECK_VISIBLE | CHECK_READONLY | CHECK_ATTACH) ||
+            /* L10N: CHECK_ACL */
```

Comment:
I think we should move the comment above the conditional, it makes it easier to see the entire conditional.